include/vrv/layerelement.h

```diff
@@ -29,6 +29,9 @@ class MeterSig;
 class Staff;
 class StaffAlignment;
 
+// Helper enums
+enum StaffSearch { ANCESTOR_ONLY = 0, RESOLVE_CROSSSTAFF };
```

Comment:
Suggest RESOLVE_CROSS_STAFF
As cross staff is two words.

Reply:
addressed here https://github.com/rism-digital/verovio/compare/17250a23c6b8...6a58106de5a2